Teacher Workout/Features/Authentication/SignUp/ViewModel/SignUpViewModel.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+class SignUpFormViewModel: ObservableObject {
```

Comment:
@vladstanescu94 Could this file be a duplicate after a renaming? I don't find it on the project, only on disk and looks the same as `SignUpFormViewModel.swift`